Assets/Scripts/UI/CurrentTurnText.cs

```diff
@@ -0,0 +1,16 @@
+using TMPro;
+using UnityEngine;
+
+namespace UI
+{
+    public class CurrentTurnText : MonoBehaviour
+    {
+        [SerializeField]
+        private TMP_Text labelText;
```

Comment:
I'm not sure if this is technically more correct than `TextMeshProUGUI`, but that's what the teacher in my unity class did so i copied him

Reply:
TextMeshProUGUI is used when working with canvas elements. TextMeshPro is used for meshes in 3D world space.

Reply:
It was setup with a canvas so it should use `TextMeshProUGUI ` right?